docs/docs/learn/introduction/gateway/index.md

```diff
@@ -0,0 +1,64 @@
+1. The relayer monitors the Bitcoin chain and sends the LP's wrapped BTC to the user after the txid is seen on Bitcoin. Specifically, the relayer submits a Merkle proof of the user's Bitcoin tx to an [on-chain light client](/learn/builder-guides/relay) for trustless verification of the user's intent. It accomplishes this by requiring that the recipient's EVM address, sequence of smart contract calls, and other order data exactly match the hash in the `OP_RETURN` of the user's Bitcoin transaction.
+
+### Liquidity Provider (LP) Flow
+
+1. An LP asks the relayer to deploy a new Gateway contract, which functions as an escrow for their funds. This is permissioned at the moment because BOB pays the transaction's gas. See the [Trust Assumptions](#trust-assumptions) section below for more information.
+2. The LP deposits wrapped Bitcoin (e.g. WBTC, tBTC, FBTC) in their Gateway contract.
+3. The LP can only withdraw their funds or update their swap fees after a delay so that the relayer has time to finish open orders. The relayer will not accept new orders during this delay until reset.
+
+## Trust Assumptions
+
+While development for BOB Gateway is advancing rapidly, there are still some training wheels in place that introduce additional trust assumptions to the architecture. These are going to be addressed and replaced with trustless or trust-minimized equivalents as the product matures.
+
+- Liquidity providers (LPs) receive BTC directly from users on Bitcoin mainnet. As a result, The off-chain relayer cannot interfere with funds received by the LPs.
+- The relayer is currently necessary to complete the user's intent (e.g. by submitting a Merkle proof of the BTC transaction to the on-chain light client, unlocking the LP's wrapped BTC). Since users cannot submit the proofs themselves yet, user funds are "stuck" whenever the relayer is offline. Users will be able to submit their own BTC proofs after we transition Gateway use a full Bitcoin light client and add functionality to protect LPs from potentially malicious strategy contracts.
+- At the moment there is only one relayer. In addition to its other functionality, this relayer pays gas on the user's behalf so that the user doesn't need to make a transaction on BOB to complete the bridging process. Since a malicious actor could create a strategy designed to take advantage of the relayer (e.g. spend all of the funds available for gas fees), the process of adding new strategies to Gateway as well as decentralizing the relayer role are restricted by the BOB team until Gateway is upgraded as described above.
```

Comment:
On more thing to add and link is the security of the light Bitcoin light client. Instead of submitting every Bitcoin block header to the relay (a full Bitcoin light client), the light client only checks that submitted Bitcoin headers meet the current or previous difficulty. While this reduces on-chain cost, it adds a level of trust on the relayer as the cost to attack the system is lower for the light Bitcoin light client. See details on cost here: https://github.com/bob-collective/bob?tab=readme-ov-file#security-light-relay

Reply:
Very cool! I love Fermi estimations like that. I'll add [that section](https://github.com/bob-collective/bob?tab=readme-ov-file#security-light-relay) to the doc tomorrow.